eDNAExplorer_Alpha.R

```diff
@@ -15,10 +15,8 @@ require(htmlwidgets)
 require(plotly)
 require(jsonlite)
 
-Sys.setenv("AWS_ACCESS_KEY_ID" = "<insert key>",
```

Comment:
@levisimons I removed these as I assume they will be set on the command line and I dont wanna hard code them in the script